libs/taskNew.js

```diff
@@ -172,26 +185,48 @@ module.exports = {
                             try {
                                 const body = JSON.parse(data);
                                 fs.unlink(bodyFile, (err) => {
-                                    if (err) cb(err);
-                                    else cb(null, body);
+                                    if (err) cb(err)
+                                    else cb(null, body)
                                 });
                             } catch (e) {
                                 cb(new Error("Malformed body.json"));
                             }
                         }
                     });
                 },
-                (cb) => fs.readdir(srcPath, cb),
```

Comment:
Should not be deleted. Sending a file and sending an s3 path should work together.